src/platform/user/profile/vet360/actions/transactions.js

```diff
@@ -240,7 +240,7 @@ export const validateAddress = (
 
     if (!confirmedSuggestions.length && validationKey) {
       // if there are no confirmed suggestions and user can override, fall back to submitted address
-      selectedAddress = payload;
+      selectedAddress = userEnteredAddress.address;
```

Comment:
Payload was coming directly from user input - which was missing some key information.  At the beginning of this thunk, we format this userEnteredAddress object to have the correct data / shape.  I am using the reference to this object to update state instead.